opmlexport/opmlexport.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Name: OPML Export
+ * Description: Export user's RSS/Atom contacts as OPML
+ * Version: 1.0
+ * Author: Fabio Comuni <https://social.gl-como.it/profile/fabrixxm>
+ * License: 3-clause BSD license
+ */
+
+use Friendica\DI;
+use Friendica\App;
+use Friendica\Core\Hook;
+use Friendica\Core\Logger;
+use Friendica\Network\HTTPException;
+use Friendica\Database\DBA;
+use Friendica\Core\Renderer;
+
+function opmlexport_install() {
+    Hook::register("addon_settings",        __FILE__, "opmlexport_addon_settings");
+    Hook::register("addon_settings_post",   __FILE__, "opmlexport_addon_settings_post");
+    Logger::log("installed opmlexport Addon");
```

Comment:
Code standards: Please normalize quotes throughout the file, preferably single quotes when double quotes aren't needed.
```suggestion
    Hook::register('addon_settings',        __FILE__, 'opmlexport_addon_settings');
    Hook::register('addon_settings_post',   __FILE__, 'opmlexport_addon_settings_post');
    Logger::log('installed opmlexport Addon");
```